### ISSUE TITLE
Unhandled exception: when trying to download mod "mutation changes"

### ISSUE THREAD

**Issue by Too-DAMN-Much** [remyroy/CDDA-Game-Launcher#90]
* Description: [Enter what you did and what happened]
* Version: 1.3.5
* OS: Windows-10-10.0.14393 (64-bit)
* Type: `<class 'rarfile.NotRarFile'>`
* Value: Not a Rar archive: C:\Users\TOO-DA~1\AppData\Local\Temp\CDDA Game Launcher\newmod-35d8a798\Mutation changes.rar
* Traceback:
```
  File "cddagl\ui.py", line 6846, in download_http_finished
  File "site-packages\rarfile.py", line 491, in __init__
  File "site-packages\rarfile.py", line 760, in _parse
  File "site-packages\rarfile.py", line 772, in _parse_real

```

**Comment by Too-DAMN-Much** [comment]
https://www.dropbox.com/s/mtjwfh0or5tqegb/Mutation%20changes.rar?dl=1 

points to 404 error.

**Comment by remyroy** [comment]
I guess the mod was removed. I'll fix this quickly tomorrow.

**Comment by Too-DAMN-Much** [comment]
https://www.dropbox.com/s/5zpxoppmorlqc1t/Mutation%20changes.rar?dl=0

appears to be the correct download link, not sure where the launcher looks for them.

**Comment by remyroy** [comment]
The author just updated (Nov 13) his modpack and deleted the old download link. I'll check for various HTTP error codes in the download process and I'll update his mod link.

**Comment by remyroy** [comment]
This should be fixed with 6a9149d348452513bfdfb80f4295d37ebb6681b7 . It will be part of the next release.
